doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -147,18 +157,10 @@ If the value equals :c:enumerator:`NRF_CLOUD_FOTA_MODEM`, the application can op
 
 Building FOTA images
 ====================
-The |NCS| build system places output images in the :file:`<build folder>/zephyr` folder.
-
-If :kconfig:`CONFIG_BOOTLOADER_MCUBOOT` is set, the build system creates the file :file:`dfu_application.zip` file containing files :file:`app_update.bin` and :file:`manifest.json`.
-Further, if :kconfig:`CONFIG_IMG_MANAGER` and :kconfig:`CONFIG_MCUBOOT_IMG_MANAGER` are also set, then the application will be able to process FOTA updates.
-
-The :file:`app_update.bin` file is a signed version of your application.
-The signature matches to what MCUboot expects, so it allows this file to be used as an update.
-The build system creates a :file:`manifest.json` file using information in the :file:`zephyr.meta` output file.
-This includes the Zephyr and |NCS| git hashes for the commits used to build the application.
-If your working tree contains uncommitted changes, the build system adds the suffix ``-dirty`` to the relevant version field.
+The build system will create the files :file:`dfu_application.zip` and/or :file:`dfu_mcuboot.zip` for a properly configured application.
+See :ref:`app_build_system` for more information about FOTA zip files.
```

Comment:
Shouldn't the link target to app_build_fota instead?

Reply:
yes